src/database/migrations/20191225105610-create-age.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('Ages', {
```

Comment:
Let's hold off on creating this table until the requirements around age are more clearly defined.